index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
```

Comment:
It's a good idea to add a title here, so we can see the title in the tab bar/title bar of the browser window.